front/app/containers/IdeasShowPage/index.tsx

```diff
@@ -88,9 +91,15 @@ const IdeasShowPage = memo<Props>(({ idea }) => {
     return (
       <IdeaNotFoundWrapper>
         <p>
-          <FormattedMessage {...messages.noResultsFound} />
+          <FormattedMessage {...messages.sorryNoAccess} />
         </p>
-        <Button linkTo="/ideas" text={goBackToListMessage} icon="arrow-back" />
+        <StyledSignInButton
+          linkTo="/sign-up"
+          text={<FormattedMessage {...messages.signUp} />}
+        />
+        <Link to={`/sign-in`}>
```

Comment:
Tiny thing: I think there's suppposed to be a linter rule about unnecessary use of backticks, but I guess it's not being triggered if it's inside the JSX part. Maybe just doing `'/sign-in'` would be better but again, super nitpicky comment so feel free to ignore

Reply:
Good catch, I think I copied a string from another component that required interpolation. I'll change it to normal quotes for consistency's sake